### PR TITLE
Updating multicloud user guide for Isabela release

### PR DESCRIPTION
**Pull request by PravinRanjan10**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md [hunk 1]
```diff
@@ -68,13 +68,24 @@ SODA Object lifecycle management feature specific UI can be accessed via
...
 
 ### Create  new backend  using remote cloud bucket
 Click on (+) for registering a storage backend
 
+![multi-cloud backend image  ](register-backend-0.png?raw=true)
+
+and then provide valid region, endpoint, Access and Secret Key parameters
+
+##### Note:
+For cloud vendors like Aws, Azure, Huawei and Gcp remote cloud buckets need not be created manually by user prior to backend registration. Remote cloud buckets can be created from SODA for these cloud vendors
```
Comment (anvithks) [inline content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md:83]: nit: Aws -> AWS, Gcp -> GCP

Comment (rajat-soda) [inline content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md:83]: Aws -> AWS , Gcp -> GCP

**Review by anvithks — APPROVED** [review]
LGTM.
Please check vendor names and correct them as per comments.

File: content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md (hunk 1 above)
Comment (PravinRanjan10) [reply content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md]: done. Thanks

Comment (PravinRanjan10) [reply content/guides/user-guides/multi-cloud/object-lifecycle-management/_index.md]: done. Thanks

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by anvithks — APPROVED** [review]
LGTM

**Pull request merged**